deepchem/data/data_loader.py

```diff
@@ -944,7 +944,7 @@ def __init__(self,
   def create_dataset(self,
                      input_files: OneOrMany[str],
                      data_dir: Optional[str] = None,
-                     shard_size: Optional[int] = None) -> DiskDataset:
+                     shard_size: int = 0) -> DiskDataset:
```

Comment:
8192 is the default we've been using elsewhere

Reply:
Do a lot of FASTA files have more than 8192 sequences?

I suppose that is sensible.